solidity/contracts/StaticOracle.sol

```diff
@@ -173,6 +212,24 @@ contract StaticOracle is IStaticOracle {
     return OracleLibrary.getQuoteAtTick(_weightedTick, _baseAmount, _baseToken, _quoteToken);
   }
 
+  function _quoteOffsetted(
+    uint128 _baseAmount,
+    address _baseToken,
+    address _quoteToken,
+    address[] memory _pools,
+    uint32 _period,
+    uint32 _offset
+  ) internal view returns (uint256 _quoteAmount) {
+    require(_pools.length > 0, 'No defined pools');
+    OracleLibrary.WeightedTickData[] memory _tickData = new OracleLibrary.WeightedTickData[](_pools.length);
+
+    for (uint256 i; i < _pools.length; i++) {
+      (_tickData[i].tick, _tickData[i].weight) = OracleLibraryPlus.consultOffsetted(_pools[i], _period, _offset);
```

Comment:
Since we are deleting `_quote`, could we handle the case where `period == 0` here?